tests/system/test_cluster/test_agent_key_polling/test_agent_key_polling.py

```diff
@@ -26,11 +26,11 @@ def configure_environment(host_manager):
                            src_path=os.path.join(os.path.dirname(os.path.abspath(__file__)), 'files/fetch_keys.py'),
                            dest_path='/tmp/fetch_keys.py')
     host_manager.apply_config(os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data/config.yml'),
-                              clear_files=[os.path.join(WAZUH_LOGS_PATH, 'ossec.log')],
+                              clear_files=[os.path.join(WAZUH_LOGS_PATH, 'wazuh.log')],
                               restart_services=['wazuh'])
     host_manager.add_block_to_file(host='wazuh-master', path='/var/ossec/etc/client.keys', replace='NOTVALIDKEY',
                                    after='wazuh-agent2 any ', before='2\n')
-    host_manager.clear_file(host='wazuh-agent2', file_path=os.path.join(WAZUH_LOGS_PATH, 'ossec.log'))
+    host_manager.clear_file(host='wazuh-agent2', file_path=os.path.join(WAZUH_LOGS_PATH, 'wazuh.log'))
```

Comment:
Same as previous comment

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/97380191165ffd7168ca42c7a8f98b6e9c177883